app/models/highlight.rb

```diff
@@ -28,6 +28,7 @@ class Highlight < ApplicationRecord
 
   scope :by_source_type, ->(source_type) { where(source_type: source_type) }
   scope :by_color, ->(color) { where(color: color) }
+  scope :by_colors, ->(colors) { where(color: colors) }
```

Comment:
do we still need the by_color scope?

Reply:
Maybe not but I think we could leave it.